react/features/calendar-sync/middleware.ts

```diff
@@ -71,6 +71,9 @@ MiddlewareRegistry.register(store => next => action => {
  * @returns {void}
  */
 function _maybeClearAccessStatus(store: IStore, { appState }: { appState: string; }) {
-    appState === 'background'
-        && store.dispatch(setCalendarAuthorization(undefined));
+
+    // Because this is shared, on web appState is always undefined,
+    // meaning that it is never active
+    return navigator.product === 'ReactNative' && (appState !== 'active'
```

Comment:
I think in this case we want the previous behavior. There is no point in resetting the permission so often.